crypto/property/property_err.c

```diff
@@ -25,7 +25,7 @@ static const ERR_STRING_DATA PROP_str_reasons[] = {
     {ERR_PACK(ERR_LIB_PROP, 0, PROP_R_NOT_A_DECIMAL_DIGIT),
     "not a decimal digit"},
     {ERR_PACK(ERR_LIB_PROP, 0, PROP_R_NO_MATCHING_STRING_DELIMETER),
-    "no matching string delimeter"},
+    "no matching string delimiter"},
```

Comment:
This file is autogenerated, the string text is generated from the symbol, `PROP_R_NO_MATCHING_STRING_DELIMETER`, which already contains the typo.

If I'm not mistaken, the constant can be renamed, because it was introduced only recently in the master branch.

Reply:
I have applied the fix for the constant as well.